src/views/templates/template-details/TemplateDetailsView.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useEffect } from 'react';
+import csx from './TemplateDetailsView.scss';
+import { Button } from 'shared/ui';
+
+import EditIcon from '@material-ui/icons/Edit';
+import StarBorderIcon from '@material-ui/icons/StarBorder';
```

Comment:
Create segments from imports. At top add all core imports like react-router, react. Add enter and then material ui imports, add enter and shared imports, add enter and move csx imports from top to bottom. You can check already created files.